drivers/video/video_mcux_csi.c

```diff
@@ -126,7 +126,6 @@ static int video_mcux_csi_set_fmt(const struct device *dev, enum video_endpoint_
 		return -EINVAL;
 	}
 
-	data->pixelformat = fmt->pixelformat;
```

Comment:
At least within this commit, isn't `video_mcux_csi_get_fmt` still using `data->pixelformat`? Might be missing something here.